package.json

```diff
@@ -28,6 +28,7 @@
     "css-select": "~2.1.0",
     "dom-serializer": "~1.1.0",
     "entities": "~2.0.2",
+    "events": "^3.2.0",
```

Comment:
`events` isn't used anywhere in this module. Let's add this to the module actually pulling in the dependency, or have users add it to their own `package.json`s.

I know that `htmlparser2` currently depends on it, but that will be fixed before the next cheerio release.

Reply:
I just removed the package "events"